packages/docs/developer-resources/contractkit/odis.md

```diff
@@ -18,19 +18,19 @@ Both methods require authentication to the ODIS server, which can be performed b
 You may use the `EncryptionKeySigner` for your `AuthSigner` by passing in the raw private key:
 
 ```ts
-  const authSigner: AuthSigner = {
-    authenticationMethod: OdisUtils.Query.AuthenticationMethod.ENCRYPTION_KEY,
-    rawKey: privateDataKey,
-  }
+const authSigner: AuthSigner = {
```

Comment:
these are probably just auto formatting changes but wanted to point out in case they're not